ios/FluentUI/Badge Field/BadgeView.swift

```diff
@@ -187,11 +137,148 @@ open class BadgeView: UIView {
+    open var labelTextColor: UIColor? {
+        get {
+            if let customLabelTextColor = _labelTextColor {
+                return customLabelTextColor
+            }
+            
+            switch style {
+                case .default:
+                    if let window = window {
+                        return Colors.primary(for: window)
+                    }
+                case .warning:
+                    return Colors.Badge.textWarning
+                case .error:
+                    return Colors.Badge.textError
+            }
+            return nil
+        }
+        set {
+            if labelTextColor != newValue {
+                _labelTextColor = newValue
+                updateColors()
+            }
+        }
+    }
+    
+    private var _selectedLabelTextColor: UIColor?
+    open var selectedLabelTextColor: UIColor {
+        get {
+            if let customSelectedLabelTextColor = _selectedLabelTextColor {
+                return customSelectedLabelTextColor
+            }
+            
+            switch style {
+                case .default:
+                    return Colors.Badge.textSelected
+                case .warning:
+                    return Colors.Badge.textWarningSelected
+                case .error:
+                    return Colors.Badge.textErrorSelected
+            }
+        }
+        set {
+            if selectedLabelTextColor != newValue {
+                _selectedLabelTextColor = newValue
+                updateColors()
+            }
+        }
+    }
+    
+    private var _disabledLabelTextColor: UIColor?
+    open var disabledLabelTextColor: UIColor? {
+        get {
+            if let customDisabledLabelTextColor = _disabledLabelTextColor {
+                return customDisabledLabelTextColor
+            }
+            return style == .default ? Colors.Badge.textDisabled : (isActive ? self.selectedLabelTextColor : self.labelTextColor)
+        }
+        set {
+            if disabledBackgroundColor != newValue {
+                _disabledLabelTextColor = newValue
+                updateColors()
+            }
+        }
+    }
+    
+    private var _backgroundColor: UIColor?
+    open override var backgroundColor: UIColor? {
+        get {
+            if let customBackgroundColor = _backgroundColor {
+                return customBackgroundColor
+            }
+            
+            switch style {
+                case .default:
+                    if let window = window {
+                        return Colors.primaryTint40(for: window)
+                    }
+                case .warning:
+                    return Colors.Badge.backgroundWarning
+                case .error:
+                    return Colors.Badge.backgroundError
+            }
+            return nil
+        }
+        set {
+            if backgroundColor != newValue {
+                _backgroundColor = newValue
+                updateColors()
+            }
+        }
+    }
+    
```

Comment:
Do you have swiftlint installed? It seems like there are a lot of formatting violations:

- Trailing Whitespace Violation: Lines should not have trailing whitespace. (trailing_whitespace)

- Switch and Case Statement Alignment Violation: Case statements should vertically align with their enclosing switch statement. (switch_case_alignment)